install/wazuh-install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 community-scripts ORG
+# Author: Omar Minaya
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://wazuh.com/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+    sudo \
+    mc \
+    curl
+msg_ok "Installed Dependencies"
+
+# Fetching the latest Wazuh version
+msg_info "Fetching Latest Wazuh Version"
+RELEASE=$(curl -s https://api.github.com/repos/wazuh/wazuh/releases/latest | grep '"tag_name"' | awk -F '"' '{print substr($4, 2)}')
```

Comment:
This returns 4.11.1 now, when you try to download the wazuh-installer.sh with version 4.11.1 that file does not exist.

I think this should fix it:
```suggestion
RELEASE=$(curl -s https://api.github.com/repos/wazuh/wazuh/releases/latest | grep '"tag_name"' | awk -F '"' '{print substr($4, 2, length($2)-4)}')
```